src/components/location/Location.js

```diff
@@ -7,12 +7,33 @@ class GoogleMap {
 	  this.position = position;
     this._googleMapKey = 'AIzaSyBApIxlKjCG4nPYE2GnZa6UltnP8JqMdgA';
     this.zoom = 15;
-    this.checkPosition();
+    this.setPosition();
   }
 
-  async checkPosition() {
-    if(this.position instanceof Function) {
-      this.position = await this.position();
+  async setPosition() {
+    if (!this.position) {
+      this.position = await this.getLocation();
+    } else {
+      this.position = position
```

Comment:
what is ` = position`? Where does it come from?

Reply:
it is additional parameter (object with latitude and longitude)
in order to have possibility generete map by given coordinates